<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QScanOrganView</class>
 <widget class="QWidget" name="QScanOrganView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QScanOrganView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 16px &quot;Microsoft YaHei&quot;;
	color: #B4C4D9;
	background-color: rgba(0,0,0,0)
}
QLabel{
	background-color: rgba(0,0,0,0);

}
QSlider{
		background-color: rgba(0,0,0,0)
}


QCheckBox
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
	background-color: rgba(0,0,0,0)
}
QCheckBox::indicator
{
width: 20px;
height: 16px;
}
QCheckBox::indicator:unchecked
{
image: url(:/CArmWorkStation/ImageSource/Patient/uncheck.png);
}

QCheckBox::indicator:checked
{
image: url(:/CArmWorkStation/ImageSource/Patient/check.png);
}

QRadioButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color: #B4C4D9;
spacing:10;
background-color: rgba(0,0,0,0)
}
  QRadioButton::indicator {
      width: 16px;
      height: 16px;
  }

  QRadioButton::indicator::unchecked {
      image: url(:/SysConfig/ImageSource/SysConfig/RadioButtom_uncheck.png);
  }
/*
  QRadioButton::indicator:unchecked:hover {
      image: url(:/images/radiobutton_unchecked_hover.png);
  }

  QRadioButton::indicator:unchecked:pressed {
      image: url(:/images/radiobutton_unchecked_pressed.png);
  }
*/
  QRadioButton::indicator::checked {
      image: url(:/SysConfig/ImageSource/SysConfig/RadioButtom_check.png);
  }
/*
  QRadioButton::indicator:checked:hover {
      image: url(:/images/radiobutton_checked_hover.png);
  }

  QRadioButton::indicator:checked:pressed {
      image: url(:/images/radiobutton_checked_pressed.png);
  }
*/</string>
  </property>
  <widget class="QNavHView" name="Scan_mode" native="true">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>266</y>
     <width>360</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>245</y>
     <width>722</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background:rgba(255,255,255,0.1);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_97">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>101</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backround-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>FOV视野大小</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Scan_defaultOrgan">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>62</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backround-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>设定为默认部位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_98">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>147</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 14px;
backround-color:rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>左右调节</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_99">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>201</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

	font:14px;
backround-color:rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>上下调节</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>137</y>
     <width>217</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{

	background-color: rgba(118,133,154,0.2);
}
QLabel{
	background-color: rgba(0,0,0,0);

}</string>
   </property>
   <widget class="QLineEdit" name="Scan_upDownAdjust">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>217</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

	background:rgba(118,133,154,0.2);
	border:1px solid rgba(74,82,90,1);

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>8</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>191</y>
     <width>217</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{

	background-color: rgba(118,133,154,0.2);
}
QLabel{
	background-color: rgba(0,0,0,0);

}</string>
   </property>
   <widget class="QLineEdit" name="Scan_leftRightAdjust">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>217</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{

	background:rgba(118,133,154,0.2);
	border:1px solid rgba(74,82,90,1);

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>8</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="ScanModeStacked">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>762</width>
     <height>604</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	font: 16px &quot;Microsoft YaHei&quot;;
	color: #B4C4D9;
	background-color: rgba(0,0,0,0)
}</string>
   </property>
   <widget class="QScanModeView" name="page"/>
   <widget class="QScanModeView" name="page_2"/>
   <widget class="QScanModeView" name="page_3"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QNavHView</class>
   <extends>QWidget</extends>
   <header>QNavigationView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QScanModeView</class>
   <extends>QWidget</extends>
   <header>QScanModeView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
